src/test/java/io/managed/services/test/billing/BillingModelTest.java

```diff
@@ -95,7 +95,7 @@ public void testAutomaticallyPickSingleCloudAccount() {
             kafka = KafkaMgmtApiUtils.createKafkaInstance(kafkaMgmtApi, payload);
             log.debug(kafka);
             assertEquals(kafka.getBillingModel(), "marketplace");
-            assertEquals(kafka.getMarketplace(), "aws");
+            assertEquals(kafka.getMarketplace(), Environment.CLOUD_PROVIDER);
             assertEquals(kafka.getBillingCloudAccountId(), Environment.STRATOSPHERE_SCENARIO_1_AWS_ACCOUNT_ID);
```

Comment:
The same as previous comment,  please do the same for rest of marketplace changes as well.